02_deployHono/main.tf

```diff
@@ -66,7 +90,7 @@ data "kubectl_path_documents" "ambassador_mappings" {
 }
 
 resource "kubectl_manifest" "ambassador_manifest" {
-  depends_on = [module.container_deployment]
+  depends_on = [module.hono]
```

Comment:
Ambassador, Jaeger and kube_prometheus_stack depends are also needed here

Reply:
Why?

Reply:
Those manifests are Ambassador specific.